### DISCUSSION TITLE
再変換ができない

### DISCUSSION THREAD

**Discussion by Hackgets** [google/mozc#870] [Q&A]
## Description
Zorin OS 17で再変換が出来ません。16では出来たのですが、再変換に割り当てられたキーを入力してもうまく行きません。Zorin OS 17はUbuntu 20.04LTSをベースとしています。ちなみに、変換中の文節の色が変わらないという不具合もありますが、これはWaylandを使っていると起きます。

## Steps to reproduce
1. キー設定をMS-IMEにし、入力後の文字列を選択して「変換」キーを押す。

## Expected behavior
正常に再変換が行われる。

## Actual behavior
一文字目だけが再変換される、文字列が消える、あるいは何も起こらない。

## Screenshots
![Screenshot from 2024-02-02 16-25-16](https://github.com/google/mozc/assets/102782728/256adac5-886b-4565-8d0e-4d93735aa072)
![Screenshot from 2024-02-02 16-25-23](https://github.com/google/mozc/assets/102782728/c5e9d842-597a-4ea2-98f8-57a3e88c4cf9)

## Version or commit-id
Mozc-2.26.4220.102+24.8.oss

## Environment
 - OS: Zorin OS 17 Core
 - Ibus 1.16.7

## Investigations
 - Whether this issue happens on both Ibus and other IMF (e.g. Fcitx, uim).
   + N/A
 - Whether this issue happens on other IMEs (e.g. Anthy, SKK).
   + N/A
 - What applications this issue happens on (e.g. Chromium, gedit).
   + gedit 41.0, Firefox 122.0
 - What applications this issue does not happen on (e.g. Chromium, gedit).
   + not-found

**Comment by konomu** [comment]
私もこの問題に逢いました。こちらはX11を使っていますので、Waylandだけの問題じゃないと思います。キー設定をMS-IMEににしなくても再変換はできません。再変換機能を別のキーに設定しても何も起こらない。
